erpc/networks_test.go

```diff
@@ -5908,3 +6026,136 @@ func setupTestNetwork(t *testing.T, ctx context.Context, upstreamConfig *common.
 
 	return network
 }
+
+func setupTestNetworkWithFullNodeUpstream(t *testing.T, ctx context.Context, upstreamConfig *common.UpstreamConfig, networkConfig *common.NetworkConfig) *Network {
```

Comment:
Should we have 2 upstreams here (one full one archive) to properly that full nodes are skipped?

If we use upstream then we should expect the new "error" we created

Reply:
agreed. created a flexible test fixture builder for nodes types with `full`, `archive` and `unknown` type.